snmp/datadog_checks/snmp/data/default_profiles/vertiv-watchdog.yaml

```diff
@@ -0,0 +1,74 @@
+extends:
+  - _vertiv.yaml
+sysobjectid:
+  - 1.3.6.1.4.1.21239.5.1 # watchdog100
+metrics:
+  - MIB: VERTIV-BB-MIB
+    table:
+      name: internalTable
+      OID: 1.3.6.1.4.1.21239.5.1.2
+    symbols:
+      - OID: 1.3.6.1.4.1.21239.5.1.2.1.5
+        name: vertiv.internalTemp
+      - OID: 1.3.6.1.4.1.21239.5.1.2.1.6
+        name: vertiv.internalHumidity
+      - OID: 1.3.6.1.4.1.21239.5.1.2.1.7
+        name: vertiv.internalDewPoint
+    metric_tags:
+      - column:
+          OID: 1.3.6.1.4.1.21239.5.1.2.1.2
+          name: vertiv.internalSerial
+        tag: vertiv_internal_serial
+#      - column:
+#          OID: 1.3.6.1.4.1.21239.5.1.1.7.0
+#          name: vertiv.temperatureUnits
+#        tag: vertiv_temperature_units
+#        mapping:
+#          0: fahrenheit
+#          1: celsius
+# TODO : Add this tag back when tagging by scalar symbol is implemented
```

Comment:
WDYT of this feature ? Should I create a ticket for it ?

Reply:
Yes, please create a ticket for it :) 

If I understand correctly, we want to tag table metrics with tag from a scalar OID value.

Reply:
Exactly